content/contributors/projects/_index.md

```diff
@@ -529,6 +529,18 @@ Sandbox Projects
 -	**License:** [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/)
 -	**Legal Requirements:** [DCO](https://developercertificate.org/)
 
+### Meshery
+
+*"Meshery is the open source, service mesh management plane that enables the adoption, operation, and management of any service mesh and their workloads."* - [Meshery](https://meshery.io)
+
+-	**Project Repository:** https://github.com/meshery
+-	**Contributor Guide:** [docs.meshery.io/project/contributing](https://docs.meshery.io/project/contributing)
+-	**Chat:** Slack: `#tinkerbell` in [slack.layer5.io](https://slack.layer5.io)
```

Comment:
```suggestion
-	**Chat:** Slack: `#meshery` in [slack.layer5.io](https://slack.layer5.io)
```
Small change, I think it was meant to be the meshery channel^^:;;

Other than that, everything looks good 👍